### PR TITLE
Adding filters page

### PR DESCRIPTION
**Pull request by efi411**

### CODE REVIEW

File: screens/FiltersScreen.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/******************************************
+ *  Author : Efi
+ *  Created On : Thu April 9 2020
+ *******************************************/
+
+import React, { useState } from "react";
+import { StyleSheet, View, Switch } from "react-native";
+import DefaultText from "../components/DefaultText";
+import LineText from "../components//LineText";
```
Comment (gilrPixellot) [inline screens/FiltersScreen.js:9]: why two // ?

File: screens/FiltersScreen.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  const [isVegetarian, setIsVegetarian] = useState(false);
+  const [isLactosIntolerance, setIsLactosIntolerance] = useState(false);
+  const [isdairy, setIsdairy] = useState(false);
+  const [isMeat, setIsMeat] = useState(false);
+  const [isDessert, setIsDessert] = useState(false);
+
+  const getFilterItem = (text, isOn, setSwitch) => {
+    return (
+      <View style={styles.filterContainer}>
+        <DefaultText style={styles.text}>{text}</DefaultText>
```
Comment (gilrPixellot) [inline screens/FiltersScreen.js:24]: maybe after other PR change the font?

File: screens/FiltersScreen.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  screen: {
+    padding: 30,
+    flex: 1,
+  },
+  filterContainer: {
+    marginVertical: 15,
+    flexDirection: "row",
+    justifyContent: "space-between",
+  },
+  title: {
```
Comment (gilrPixellot) [inline screens/FiltersScreen.js:65]: again- font

File: screens/FiltersScreen.js (hunk 2 above)
Comment (efi411) [reply screens/FiltersScreen.js:24]: Not relevant for regular text.

File: screens/FiltersScreen.js (hunk 1 above)
Comment (efi411) [reply screens/FiltersScreen.js:9]: Done.

File: screens/FiltersScreen.js (hunk 3 above)
Comment (efi411) [reply screens/FiltersScreen.js:67]: Done.

**Pull request merged**